packages/clerk-js/src/ui/router/RouteContext.tsx

```diff
@@ -1,4 +1,3 @@
-import type { ParsedQs } from 'qs';
```

Comment:
Can we remove/uninstall the `qs` dependency from the package or is it still used somewhere in your branch? If yes, we have the chance to fix SDK-873 then 👍

Reply:
I thought about the same. Unfortunately, it's still used by `fapiClient`. I will try to handle it with a new PR as it is not related to the changes introduced here :)